zipkin-storage/elasticsearch/src/main/java/zipkin/storage/elasticsearch/InternalElasticsearchClient.java

```diff
@@ -49,7 +49,7 @@
    *
    * <p> See https://www.elastic.co/guide/en/elasticsearch/reference/current/search-request-from-size.html
    */
-  protected static final int MAX_RAW_SPANS = 10000; // the default elasticsearch allowed limit
+  protected static final int MAX_RAW_ITEMS = 10000; // the default elasticsearch allowed limit
```

Comment:
I think this is used in a subquery of spans, too, so it wouldn't be accurate to say max raw items (I can verify). Would be nice if there was a way to say "give me all"